### PR TITLE
Add more documentation for pod security standards

### PR DESCRIPTION
**Pull request by crobby**
Issue:  https://github.com/rancher/rancher/issues/40575

Related to/based on https://github.com/rancher/rancher-docs/pull/423 (ie:  that commit will disappear from this PR once it's merged)

### CODE REVIEW

**Comment by maxsokolovsky** [comment]
@crobby, @cbron, what should we do with any old references to PSPs throughout the docs?
For example, [here](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions#list-of-restricted-admin-permissions) we list Global Permissions and mention the PSP Management permission category.

Are the docs always supposed to be covering the latest Rancher release? Or do we keep those references, since some v2.7 releases support PSPs?

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 1]
```diff
@@ -2,21 +2,52 @@
...
+for the namespaces used by Rancher to allow the Rancher system pods to perform actions
+that would not otherwise be permitted.
 
-###  TODO
+* Here is a list of the namespaces that require an exemption:
+    * _kube-system, kube-public, cattle-system, cattle-alerting, cattle-logging, cattle-prometheus,
+ingress-nginx, cattle-global-data, cattle-istio, kube-node-lease, cert-manager,
+cattle-global-nt, security-scan, cattle-fleet-system, cattle-fleet-local-system,
+calico-system, tigera-operator, cattle-impersonation-system, rancher-operator-system,
+cattle-csp-adapter-system, calico-apiserver_
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:50]:
```suggestion
    * kube-system
    * `kube-public`
    * `cattle-system`
    * `cattle-alerting`
    * `cattle-logging`
    * `cattle-prometheus`
    * `ingress-nginx`
    * `cattle-global-data`
    * `cattle-istio`
    * `kube-node-lease`
    * `cert-manager`
    * `cattle-global-nt`
    * `security-scan`
    * `cattle-fleet-system`
    * `cattle-fleet-local-system`
    * `calico-system`
    * `tigera-operator`
    * `cattle-impersonation-system`
    * `rancher-operator-system`
    * `cattle-csp-adapter-system`
    * `calico-apiserver` 
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 2]
```diff
@@ -2,21 +2,52 @@
...
 
-* PSAC templates
-	* How they work, what the flow is
-	* That updating a template will affect live clusters
-	* Who can edit. Including a template management role workaround with it's own warning/caveat.
-* How to upgrade
-	* Ensure that a replacement is in place before you upgrade and PSPs are off
\ No newline at end of file
+* A sample AdmissionConfiguration with all the required exemptions to run Rancher
+can be found [here](psa-restricted-exemptions.yaml)
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:53]:
```suggestion
* Use this [sample AdmissionConfiguration](psa-restricted-exemptions.yaml) with all the required exemptions.
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by crobby** [comment]
> @crobby, @cbron, what should we do with any old references to PSPs throughout the docs? For example, [here](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions#list-of-restricted-admin-permissions) we list Global Permissions and mention the PSP Management permission category.
> 
> Are the docs always supposed to be covering the latest Rancher release? Or do we keep those references, since some v2.7 releases support PSPs?

It feels like we might need to keep them around, but I'll defer to others on this one.

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 3]
```diff
@@ -2,21 +2,52 @@
...
+clean up all configurations related to Pod Security Policies (PSPs) before you begin your upgrade.
+
+## Pod Security Admission Configuration Templates
+
+Rancher comes with two Pod Security Admission Configuration (PSAC) templates that you can assign to a cluster:
+* `rancher-privileged`:  This is the most permissive configuration and doesn't restrict the behavior of any pod.
+* `rancher-restricted`:  This policy follows current pod-hardening best practices.
+
+
+If you are an Rancher administrator or have restricted administrator privileges,
```
Comment (maxsokolovsky) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:19]: `a Rancher administrator`.

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 4]
```diff
@@ -2,21 +2,52 @@
...
+or by editing existing templates.
+
+:::caution
+If you edit an existing PSAC template while it is still in use, it *will* affect all clusters
+that have been assigned that template.
 
+:::
 
+If you want to allow users other than the Rancher administrator to manage PSAC templates,
+you can create a role/binding that grants the user all verbs
```
Comment (maxsokolovsky) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:30]: What exact verbs are we talking about?

Comment (cbron) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:30]: How about:

```suggestion
you can bind the user to a role that grants the user all verbs ("*") on... 
```

I would put the warning below directly into this section.

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 5]
```diff
@@ -2,21 +2,52 @@
...
+on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
 
+:::warning
+Any user that is bound to the above permission will be able to change the restriction levels on all managed clusters.
```
Comment (cbron) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:34]:
```suggestion
Any user that is bound to the above permission will be able to change the restriction levels on all managed clusters which use this template, including ones they have no permissions on.
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md [hunk 6]
```diff
@@ -2,21 +2,52 @@
...
 
+### Running Rancher on a local cluster that is PSA restricted by default
 
+When running Rancher on a Kubernetes cluster that is enforcing the "restricted"
+Pod Security Standard by default, you will need to add some exemptions to that policy
+for the namespaces used by Rancher to allow the Rancher system pods to perform actions
+that would not otherwise be permitted.
 
-###  TODO
+* Here is a list of the namespaces that require an exemption:
```
Comment (cbron) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md:45]: This is overlapping with https://github.com/rancher/rancher-docs/pull/423 ? Is that one going to close.

**Review by cbron — COMMENTED** [review]
> what should we do with any old references to PSPs throughout the docs?

Yes we need to keep them around, and in history for any 2.7 docs, just like 2.5 has today.

**Pull request closed without merging**